### ISSUE TITLE
Ankh issues

### ISSUE THREAD

**Issue by Ca3desnox** [00-Evan/shattered-pixel-dungeon#250]
I have the latest F-Droid version installed. 
I found two isssues:
1. I had 2 blessed ankhs and fought a monk. When I presumably should have died the game got stuck "thinking" (continuing turning circle). Closing and loading the game I found myself where I was before. All Items intact and one of the ankhs gone.

2. I continued playing with one blessed ankh and bought another which wasn't blessed when I died again. The game then asked me if I wanted to go on(I assume that should have happened in issue 1 as well). When I accepted it used the unblessed ankh and everything was gone. 

Tl:Dr:
Game uses unblessed ankh instead of blessed ankh
game "freezes" when having 2 blessed ankhs and one dies

**Comment by 00-Evan** [comment]
I'll look into this, but I've had this reported to me occasionally over the years and every time it turned out to be someone mistaking which ankhs were blessed and which weren't.

I would like to adjust ankhs in general at some point though, to make unblessed ones less terrible.

**Comment by 00-Evan** [comment]
Can't reproduce this, closing for now. Feel free to comment if this happens to you again.
